docsDoxySphinx/api/cpp_doxygen_sphinx.rst

```diff
@@ -37,10 +37,10 @@ Types
    :project: Redev
 
 
```

Comment:
I think we should consider documenting the `Partition` variant type and discuss its use with `GetRank(...)`.  Beyond that, providing an example showing how a function taking a partition object can use the `visit` function (and the `redev_variant_tools.h` header) to call either `GetRank(...)` interface would also be helpful for our future selves.